src/17-infinite-discovery/helpers.ts

```diff
@@ -60,8 +60,8 @@ export function resizeImage(
     height: number
   }
 ) {
-  const { width, height } = defaults(options, { width: 500, height: 500 })
-  return `https://d7hftxdivxxvm.cloudfront.net/?src=${src}&resize_to=fit&width=${width}&height=${height}&grow=false`
+  const { width, height } = defaults(options, { width: 224, height: 224 })
```

Comment:
This is the default resolution configured in the `CLIP` [docs](https://huggingface.co/docs/transformers/en/model_doc/clip#transformers.CLIPVisionConfig), so I have have copied that here. It is what has made the biggest difference.